style.css

```diff
@@ -147,54 +156,266 @@
   opacity: 0.3;
 }
 
-.tonic {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  color: rgb(0, 0, 82);
-  margin: 15px;
-  margin-top: 30px;
+.project-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  background-color: white;
+  box-sizing: border-box;
+  border-radius: 10px;
+  padding: 10px;
+}
+
+.work {
+  background-color: #e5e5e5;
+  width: 100%;
 }
 
-.tonic-img {
-  margin-left: 27px;
+.work-img {
   display: flex;
   flex-direction: column;
   justify-content: center;
-  height: 370px;
-  width: 370px;
+  margin: 10px 10px;
 }
 
-.tonic-text {
+.project-title {
+  margin: 5px 10px;
+  display: flex;
   flex-direction: column;
+  align-items: flex-start;
+  font-size: 42px;
+  color: rgb(0, 10, 47);
+  font-family: "Poppins", sans-serif;
+  font-weight: bold;
+  margin-top: 12px;
+  margin-left: 15;
+}
+
+.some-list {
+  margin: 5px 10px;
+  margin-left: 15px;
+}
+
+.work-info {
+  display: flex;
+  flex-direction: row;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  padding: 15px;
+}
+
+.lang-list {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 82%;
+  margin-left: -5%;
+  padding: 4px 12px;
+}
+
+.Html {
+  margin: 12px 2px;
+  padding: 5px 12px;
+  font-size: 16px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border: 2px solid #ebebff;
+  border-radius: 9px;
+  outline: none;
+}
+
+.Css {
+  margin: 12px 2px;
+  padding: 5px 10px;
+  font-size: 16px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border: 2px solid #ebebff;
+  border-radius: 9px;
+  outline: none;
+}
+
+.js {
+  margin: 12px 2px;
+  padding: 5px 10px;
+  font-size: 20px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border: 2px solid #ebebff;
+  border-radius: 9px;
+  outline: none;
+}
+
+.button {
+  margin: 1.3rem;
+  padding: 0.7rem 1.5rem;
+  font-size: 20px;
+  color: #6070ff;
+  background-color: #fff;
+  border: 2px solid blue;
+  border-radius: 7px;
+  outline: none;
+  cursor: pointer;
+}
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, it would be better to match colors for all your elements to that of the Figma design, 😎 

You may try to use ```#396DF2``` instead of using ```blue``` to replicate the[ given design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Ws4xmSpNwGoWQ4iz-0) 

eg: 
**Figma buttons**: ![image](https://user-images.githubusercontent.com/102757126/212392801-fe9e9f6f-2009-4b68-961d-688426729bfb.png)

**Your button:** ![image](https://user-images.githubusercontent.com/102757126/212392884-20dfc78c-5ae6-4925-b81b-5ffd77ecb308.png)


It would be better to check the original color from Figma to make a perfect website 🙏